server/src/test/java/org/opensearch/index/translog/transfer/TranslogTransferManagerTests.java

```diff
@@ -348,6 +367,8 @@ public void testDownloadTranslogAlreadyExists() throws IOException {
         verify(transferService).downloadBlob(any(BlobPath.class), eq("translog-23.ckp"));
         assertTrue(Files.exists(location.resolve("translog-23.tlog")));
         assertTrue(Files.exists(location.resolve("translog-23.ckp")));
+        assertTrue(remoteTranslogTransferTracker.getDownloadBytesSucceeded() > 0);
+        assertEquals(2, remoteTranslogTransferTracker.getTotalDownloadsSucceeded());
```

Comment:
The name suggests that the files already exists, but the download stats suggest that there were some files downloaded still. Could not understand what the original test intends to do and the addition we are doing here.

Reply:
As per ...

https://github.com/opensearch-project/OpenSearch/blob/f9a661cd983255253307d5d07947dc47b129ab16/server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferManager.java#L176-L188

... we remove any existing tlog files on local with a name same as the files we are intending to download from remote. And then we still go ahead with the download from remote. 

I think the original test intends to validate that we are downloading the blobs even if files with the respective names were already present on local.

What we are adding here is the assertion on the bytes captured in the stats tracker (but `assertEquals(2, remoteTranslogTransferTracker.getTotalDownloadsSucceeded());` will no longer be asserted as we are doing it on a sync level now).